pkg/controller/jobframework/integrationmanager.go

```diff
@@ -370,3 +386,33 @@ func GetMultiKueueAdapters(enabledIntegrations sets.Set[string]) (map[string]Mul
 	}
 	return ret, nil
 }
+
+func (m *integrationManager) autoEnableIntegrations(ctx context.Context, enabledFrameworks sets.Set[string]) {
```

Comment:
Probably this function should return auto enable integrations instead of adding it to set, right?